src/__tests__/fcnnls_deconv.test.js

```diff
@@ -0,0 +1,20 @@
+import fcnnls from '../fcnnls';
+
+import { X as X1, Y as Y1 } from './deconv-examples/data_example1';
+import { X as X2, Y as Y2 } from './deconv-examples/data_example2';
+import { X as X3, Y as Y3 } from './deconv-examples/data_example3';
+
+describe('fcnnls deconv-examples run', () => {
+  it('example 1', () => {
+    const result = fcnnls(X1, Y1);
+    expect(result).toBeDefined();
+  });
+  it('example 2', () => {
+    const result = fcnnls(X2, Y2, { gradientTolerance: 1e-4 });
```

Comment:
Does it crash with default gradientTolerance ?

Reply:
Yes, both example 2 and 3. 

We can make it `1e-4`, but this value will depend in the specific example.